test/case/ietf_routing/ospf_default_route_advertise/Readme.adoc

```diff
@@ -0,0 +1,34 @@
+=== OSPF Default route advertise
+==== Description
+Test advertising default route adverting, sometimes called 'redistribute origin' both
```

Comment:
- Verify instead of Test
- duplicate advertising/adverting.  

Maybe: "Verify default route advertising over OSPF, sometimes called ..."